.travis/docker_hub_push_release.sh

```diff
@@ -2,12 +2,17 @@
 # Should be run from repository's root
 # Usage: bash .travis/docker_hub_push_releases.sh
 
-if [ "$VULAS_RELEASE" == "$TRAVIS_TAG" ]; then
-    echo "$DOCKER_HUB_NARAMSIM_PASSWORD" | docker login -u "$DOCKER_HUB_NARAMSIM_USERNAME" --password-stdin
-    (cd docker && bash push-images.sh -r docker.io -p vulas -v "$TRAVIS_TAG")
+#
+# if [ "$VULAS_RELEASE" != "$TRAVIS_TAG" ]; then
```

Comment:
Why did you comment the `if` check?

Reply:
I commented it out to perform tests. I'll add it back in right now